server/src/api/discounts/discounts.service.ts

```diff
@@ -38,7 +38,6 @@ export const addDiscountCode = async (params: any): Promise<response> => {
     values: [params.code, params.amount, params.percent, params.startdate, params.enddate,
       params.tickettypeid, params.createdby, params.usagelimit, params.min_tickets, params.min_events],
   };
```

Comment:
Could you update the `addDiscountCode` method as well? `tickettypeid`->`tickettypeid_fk` and  `createdby`->`createdby_fk` are the only ones I think need renamed